packages/ui-components/src/lib/utils/hex.ts

```diff
@@ -3,3 +3,5 @@ import { toHex } from 'viem';
 export const HEX_INPUT_REGEX = /^(0x)?([0-9a-f]+)?$/;
 
 export const bigintStringToHex = (val: string) => toHex(BigInt(val));
+
+export const bigintToHex = (val: bigint) => toHex(val);
```

Comment:
what am i looking at here? this is just a renaming of the viem function?

Reply:
yes this is just a wrapper for the viem function which we don't need at all. will remove this one